### DISCUSSION TITLE
webhook通知失败

### DISCUSSION THREAD

**Discussion by lonely06** [jeessy2/ddns-go#895] [Q&A]
向teams进行消息推送时，出现了报错：context deadline exceeded (Client.Timeout exceeded while awaiting headers)
麻烦请教一下，这是什么原因

**Comment by jeessy2** [comment]
网络问题，超时

**Reply by lonely06** [reply]
出现这种情况，会有重发机制吗？目前没有看到

**Reply by jeessy2** [reply]
暂无，只有等待下次

**Reply by lonely06** [reply]
了解，感谢及时回复
